packages/ove-app-maps/README.md

```diff
@@ -0,0 +1,49 @@
+# Maps App
+
+This app supports visualisation of dynamic maps using the OVE framework. It is based on [OpenLayers](https://openlayers.org/) and supports tiled map layers (from Bing, OSM, etc), vector data described in formats such as [GeoJSON](http://geojson.org/) and custom overlays built using JavaScript libraries such as [D3.js](https://d3js.org/).
+
+The maps app depends on a *Map Layers* configuration that can be provided within the `config.json` file (either embedded or as a URL) or as an environment variable named `OVE_MAPS_LAYERS`, that points to a URL.
+
+## Application State
+
+The state of this app has a format similar to what is provided below:
+
+```json
+{
+    "center": ["-11137.70850550061", "6710544.04980525"],
+    "resolution": "77",
+    "zoom": "12",
+    "enabledLayers":["23"],
+    "scripts":["http://my.domain/single.js"]
```

Comment:
Need to explain that this can be a file that includes other js/css files, and may interact with the OpenLayers map object created by the maps app.

Reply:
There is already a comment from @davidbirchwork asking to explain how the custom layers work. I will add some instructions along those lines.